TypeScript d.ts errors with corejs@0.9.18

For all generated d.ts files in Aurelia explicitly using core-js, core js is imported as 

`import core from 'core-js'`

This throws the following TypeScript compiler error, for example

`jspm_packages/github/aurelia/templating@0.13.15/aurelia-templating.d.ts(2,10): error TS1192: External module '"core-js"' has no default export.`

Changing the first line to the following resolves the error. This change is consistent with the d.ts fetched from tsd for corejs@0.9.18

`import * as core from 'core-js';`

--- a/src/index.js
+++ b/src/index.js
@@ -1,4 +1,4 @@
-import core from 'core-js';
+import * as core from 'core-js';
 import {History} from 'aurelia-history';
 
 // Cached regex for stripping a leading hash/slash and trailing space.

